Gemfile.lock

```diff
@@ -27,7 +27,7 @@ GEM
     parallel (1.19.2)
     parser (2.7.1.4)
       ast (~> 2.4.1)
-    rack (2.2.2)
+    rack (2.2.3)
```

Comment:
Where did these changes come from?

Reply:
just from running the setup command. Should be safe